src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -59,72 +59,76 @@ export const RegistrationFormActions = ({
     }
   };
 
+  const isFirstTab = tabNumber === RegistrationTab.Description;
+  const isLastTab = tabNumber === RegistrationTab.FilesAndLicenses;
+
   return (
     <>
       <Box
         sx={{
-          mb: '1rem',
           display: 'grid',
           gridTemplateAreas: {
-            xs: "'save-next-button save-next-button' 'back-button support-button'",
-            sm: '"back-button support-button save-next-button"',
+            xs: "'back-button next-button' 'support-button save-button'",
```

Comment:
Disse skulle kanskje vært flyttet om på, slik at neste og forrige alltid kommer nederst? 🤔 
```suggestion
            xs: "'support-button save-button' 'back-button next-button'",
```
Lurer på om det er litt rart at man man scrolle forbi neste/forrige-knapp for å kunne lagre 
![bilde](https://user-images.githubusercontent.com/6313468/219580031-7a9fc054-7d9d-47c6-bb1f-a678c10f473e.png)